src/style.css

```
@@ -0,0 +1,98 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: "Avenir-Book";
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: "Avenir-Heavy";
+  font-style: normal;
+  font-weight: 400;
+}
+
+html {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+}
+
+.form {
+  position: relative;
+}
+
+.input {
+  display: flex;
+  font-family: inherit;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  margin: 20px auto;
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input-small {
+  font-family: inherit;
+  height: 42px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  padding-left: 33px;
+  line-height: 19px;
```

```suggestion
  height: 42px;
  border-radius: 4px;
  font-size: 14px;
  padding-left: 33px;
  line-height: 19px;
```
Remove styles which you already set for common `input` class

---

https://valeriiafomenko.github.io/layout_search-bar-airbnb/
https://valeriiafomenko.github.io/layout_search-bar-airbnb/report/html_report/